src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/GenerateNuGetPackageProjectContextMenuCommand.cs

```diff
@@ -2,17 +2,23 @@
 
 using System.ComponentModel.Composition;
 using Microsoft.VisualStudio.Packaging;
+using Microsoft.VisualStudio.ProjectSystem.Build;
 using Microsoft.VisualStudio.ProjectSystem.Input;
+using Microsoft.VisualStudio.Shell;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
 {
     [ProjectCommand(ManagedProjectSystemPackage.ManagedProjectSystemCommandSet, ManagedProjectSystemPackage.GenerateNuGetPackageProjectContextMenuCmdId)]
-    //[AppliesTo(ProjectCapability.Pack)]
+    [AppliesTo(ProjectCapability.Pack)]
```

Comment:
https://github.com/NuGet/Home/issues/4145 has been done. I am going to merge this PR.